fastlane/spec/actions_specs/add_git_tag_spec.rb

```diff
@@ -14,7 +14,9 @@
           add_git_tag
         end").runner.execute(:test)
 
-        expect(result).to eq("git tag -am builds/test/1337\\ \\(fastlane\\) \'builds/test/1337\'")
+        message = "builds/test/1337 (fastlane)"
+        tag = "builds/test/1337"
+        expect(result).to eq("git tag -am #{message.shellescape} #{tag.shellescape}")
```

Comment:
matching the previous comment, `'` are gone here as well and replaced with `shellescape`